src/DependencyInjection/Compiler/AddFilterTypeCompilerPass.php

```diff
@@ -35,6 +35,11 @@ public function process(ContainerBuilder $container)
             $serviceDefinition->setPublic(true); // Temporary fix until we can support service locators
 
             $types[$serviceDefinition->getClass()] = $id;
+
+            // NEXT_MAJOR: Remove the alias when dropping support for symfony 2.x
```

Comment:
This is not related to symfony 2 anymore, it's related to the AdminDoctrineOrmBundle version

Reply:
Adding a conflicting version should solve this even better

Reply:
The problem is if users are still using string names like `doctrine_orm_*`, it won't work